homeassistant/components/update/translations/bg.json

```diff
@@ -1,3 +0,0 @@
-{
```

Comment:
These deletions should still not exist in this PR

Reply:
Being "new" here - I'm not sure how to fix this screw up.  I thought this was one of the errors being tossed by CI. 

Can you run CI for me again (to see if I fixed the problem)

Reply:
I just added it back in - I figured there was a way to remove the commit, but just adding it back in seems to have done it.